src/blog/2024/07/deploying-flowfuse-with-docker.md

```diff
@@ -82,7 +84,7 @@ wget <link of the latest tar.gz release>
 
 The `flowforge.yml` file was updated to include our domain in key fields: `domain`, `base_url`, and `broker.public_url`. These adjustments ensure that instance names on Docker platforms incorporate your domain, provide accurate URLs for accessing the platform, and specify the correct URL for devices to connect to the broker if different from `broker.url`. Additionally, in the `docker-compose.yml` file, we configured `VIRTUAL_HOST` and `LETSENCRYPT_HOST` to reflect our domain.
 
-For more details on these configuration changes, refer to the [documentation](/docs/install/docker/#configuring-flowfuse).
+For more details on these configuration changes, refer to the [documentation](/docs/install/docker/configuration/#configuring-flowfuse).
```

Comment:
There is no such paragraph on https://flowfuse.com/docs/install/configuration/ like `Configuring Flowfuse`. The link should be just `/docs/install/docker/configuration.md` IMO

Reply:
it has that heading

Reply:
my bad, you were right on the docker docs there is no page that explains flowfuse configuration, the correct link should be /docs/install/configuration/#configuring-flowfuse